sw/host/opentitanlib/src/ownership/owner.rs

```diff
@@ -0,0 +1,594 @@
+// Copyright lowRISC contributors.
```

Comment:
I just realized the copyright header may have been changed. Need to check copyright linter results.

Reply:
Fixed.